### PR TITLE
Allow CSV safe decode

### PR DESCRIPTION
**Pull request by rockneurotiko**
On #11 we added consistency to the CSV decode and xls decode, but maybe someone want to use the safe decode method, this uses an option in the parser_options to execute with the safe version of CSV.decode

Usage: `Exoffice.parse(path, parser_options: [safe: true])`

### CODE REVIEW

**Review by danielRomero — APPROVED** [review]
LGTM

File: test/exoffice_test.exs [hunk 1]
```diff
@@ -33,5 +33,38 @@ defmodule ExofficeTest do
       assert is_pid(pid)
       assert Exoffice.get_rows(pid, parser) |> Enum.to_list() == expected
     end
+
+    test "decode CSV in a safe way" do
+      [{:ok, pid, parser}] =
+        Exoffice.parse("./test/test_data/test_semicolon.csv", parser_options: [safe: true, separator: ?;])
```
Comment (AlexKovalevych) [inline test/exoffice_test.exs]: Thank you for your PR, could you please add a test with invalid CSV data to make sure it will not raise in safe mode?

**Review by AlexKovalevych — CHANGES_REQUESTED** [review]

File: test/exoffice_test.exs (hunk 1 above)
Comment (rockneurotiko) [reply test/exoffice_test.exs]: Sure! Done :)

**Review by AlexKovalevych — APPROVED** [review]

**Comment by rockneurotiko** [comment]
Thanks! :heart:

**Pull request merged**